cortex/__init__.py

```diff
@@ -3,7 +3,7 @@
 import warnings
 
 from cortex.dataset import Dataset, Volume, Vertex, VolumeRGB, VertexRGB, Volume2D, Vertex2D
-from cortex import align, volume, quickflat, webgl, segment, options
+from cortex import align, volume, quickflat, webgl, segment, options, fmriprep
```

Comment:
`cortex.freesurfer` isn't imported, so `fmriprep` shouldn't be imported either I'd say.

Then again `align` and `segment` are imported, so I leave that decision to people who have a better overview